### ISSUE TITLE
Is it possible to allocate a certain balance to invest?

### ISSUE THREAD

**Issue by firepol** [DeviaVir/zenbot#1095]
Hi, I'm new to zenbot, I tried the paper mode and as I understand it the bot will invest all the currency it can find. So even if I set c.buy_pct = 10 to buy with 10% of currency, after 10 buy orders and no sell, the bot basically invested all my money.

Is it possible to set a balance allocated to th bot? E.g. if I have 1 BTC, can I allocate to the bot only 0.2 BTC for automated trading, leaving 0.8 BTC available for my manual trades or for other instances of the bot that I'd like to allocate to other cryptos?

**Comment by psheehan89** [comment]
Perfect timing, a PR (https://github.com/DeviaVir/zenbot/commit/10c23846ac9c8c0208e701c4c91b81aec4707593) merged like 3 days ago that lets you do just that with `--buy_max_amt`

**Comment by firepol** [comment]
As I understand, this option is per trade, not the total amount of currency to allocate to the bot. So if I have 10 BTC and I set buy_max_amt to 1 BTC, if the bot believes it's a good time to buy, it'll buy 10 times 1 BTC and use all my currency anyway... at least on paper mode I saw the simulator doing many buy orders like this. Also, on paper mode the bot doesn't obey this option. Does this work only on live trading for now, but not on paper mode?

So it seems to me that it's not yet possible to allocate a maximum amount of currency to the bot, if I want to achieve this I have to create different accounts, e.g. bitstamp allows to create a sub-account e.g. for bot trading. Other exchanges don't provide such an option and one has to create different accounts (not optimal)...

These options seem to be what I'm looking for:

```
    --currency_capital <amount>     for paper trading, amount of start capital in currency
    --asset_capital <amount>        for paper trading, amount of start capital in asset
```

But they seem to be only for paper trading.

It would be cool to have something like this:

```
    --currency_capital_max_amt <amount>     maximum amount of capital in currency to allocate to the bot
    --asset_capital_max_amt <amount>        maximum amount of capital in asset to allocate to the bot (e.g. if you plan selling coins you already own, instead of buying new ones)
```
